hack/lib/golang.sh

```diff
@@ -146,9 +145,11 @@ readonly KUBE_TEST_PORTABLE=(
   hack/lib
 )
 
-# Node test has built-in etcd and kube-apiserver, it can only be built on the
-# same platforms with kube-apiserver.
+# Test targets which run on the Kubernetes clusters directly, so we only
+# need to target server platforms.
+# These binaries will be distributed in the kubernetes-test tarball.
 readonly KUBE_NODE_TEST_TARGETS=(
+  cmd/kubemark
```

Comment:
Why node_test_targets? It's only on a machine from which the tests are run, to push the binary to gcr.io